cdap-docs/_common/_themes/cdap/static/version-menu.js

```diff
@@ -40,15 +40,15 @@
  * 
  * list of development versions; one current version; list of additional versions; dict of Google Custom Search Engines (gcse)
  *
- * version 0.4
+ * version 0.5
  * 
  */
 
 (function() {
   var versionsURL = 'http://docs.cask.co/cdap/';
   var versionID = 'select-version';
   var buildURL = (function(dir){
-    return versionsURL + dir + '/en/';
+    return versionsURL + dir + '/en/' + location.pathname.substr(location.pathname.lastIndexOf('/en/')+4);
```

Comment:
Is there any reason that this isn't `return versionsURL + dir +  location.pathname.substr(location.pathname.lastIndexOf('/en/'));`?